### DISCUSSION TITLE
Τα MX Linyx είναι αποδεκτά;

### DISCUSSION THREAD

**Discussion by perikarananou** [courses-ionio/sw#1237] [Q&A] [answered]
Τα [MX Linyx](https://mxlinux.org/) είναι  systemd-free βασισμένο σε Debian αλλά και antiX. Χρησιμοποιεί το Xfce σαν περιβάλλον ενώ υπάρχουν εκδώσεις  KDE και Fluxbox. Θα ήθελα να τα δοκιμάσω καθώς έχουν πολλές επιλογές και μπορείς να κάνεις πλήρες customization. Τέλος είναι ένα σχετικά πρόσφατο σύστημα το   οποίο συνεχώς ενημερώνεται. Κάποιες επιπλέων [πληροφορίες](https://distrowatch.com/table.php?distribution=mx). Ένα τέτοιο λογισμικό θα ήταν καλή επιλογή για να βαθμολογηθεί πλήρως;

**Comment by NickLitharis** [comment]
Καλησπέρα συνάδελφε, σύμφωνα με τις οδηγίες και με τον καθηγητή για το δεύτερο παραδοτέο η διανομή του Linux που θα εγκαταστήσεις θα πρέπει:
1. Να μην έχει systemD
2. Να μην έχει automatic installer

Θεωρητικά αν καλύπτεις αυτές τις προϋποθέσεις δεν υπάρχει κάποια μείωση βαθμού.

**Comment by P2019217** [comment]
Τα MXLinux δεν διαθέτουν systemsd αλλά sysVinit και πιστεύω αποδεκτή λύση

**Comment by KonstantinosTourtsakis** [comment, marked as answer]
Σκοπός του παραδοτέου είναι κυρίως να μάθεις πως γίνεται η εγκατάσταση ενός λειτουργικού συστήματος από πηγαίο κώδικα. Όπως βλέπεις στις [οδηγίες](https://courses-ionio.github.io/help/cli/) αναφέρεται πως τα BSD συστήματα βαθμολογούνται με μειωμένο βαθμό παρά το γεγονός πως δεν χρησιμοποιούν systemd. Ο λόγος είναι επειδή η εγκατάσταση είναι απλή και πιο φιλική προς τον χρήστη με αποτέλεσμα να χάνεται η γνώση και η εμπειρία που σου προσφέρει (ενδεικτικά) ένα από τα προτεινόμενα συστήματα.

Σύμφωνα με τα παραπάνω δεν προτείνω να το χρησιμοποιήσεις γιατί όπως φαίνεται στις [οδηγίες χρήστη](https://github.com/MX-Linux/mx-docs/raw/master/mxum.pdf) η εγκατάσταση γίνεται μέσα από γραφικό περιβάλλον.

**Σημείωση:** αναφέρω το γραφικό περιβάλλον επειδή γίνεται χρήση αυτού στην περίπτωση του MXLinux. Ένα σύστημα που δεν έχει γραφικό περιβάλλον (και systemd) αλλά η εγκατάσταση γίνεται μέσω κάποιου package manager επίσης **δεν** εξυπηρετεί τους σκοπούς του παραδοτέου.

**Reply by Angeloth1** [reply]
Πολύ ωραία τα λες συνάδελφε στο μόνο που διαφωνώ με βάσει το disc #1151, η χρήση γραφικού installer δεν αποτελεί πηγή προβλήματος αλλά η χρήση κάποιου συστήματος του οποίου δεν είναι επιτρεπτή η προσαρμογή του σε χαμηλό επίπεδο.

**Reply by KonstantinosTourtsakis** [reply]
Ναι, η αλήθεια είναι πως με το γραφικό περιβάλλον δεν απαντάω άμεσα στο ερώτημα "είναι αποδεκτά?" αλλά προσθέτω λίγες πληροφορίες που θα βοηθήσουν τους συναδέλφους που μπορεί να θεωρήσουν πως μπορούν να βαθμολογηθούν με άριστα στο παραδοτέο αυτό κάνοντας χρήση τέτοιων συστημάτων.
